recipes/radis/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "radis" %}
+{% set version = "0.13.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 650ef57b6f6b96232e4282564b613650538ff914268e71b67a4ea846c9a8d746
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```

Comment:
Upstream seem to have a compiled extension and other other dependencies that you missed there. Can you try to generate the recipe with `grayskull pypi --strict-conda-forge radis` instead? It is much less error prone and easier for us to review.